Elements.css

```diff
@@ -1,117 +1,141 @@
 #outside_container {
-  position: fixed;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background: url(background.jpg);
-  z-index: -1;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: url(background.jpg);
+    z-index: -1;
 }
 
 #container {
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-
-  z-index: 1;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    margin-bottom: 50px;
+    overflow: auto;
 }
 
 .container_2 {
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-
-  z-index: 1;
+    border: 2px solid #000000c5;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    background: #c0d698c5;
+    overflow: auto;
+    margin-bottom: 50px;
 }
 
 #avatar_container {
-  position: relative;
-  top: 30%;
-  left: 50%;
-  transform: translate(-50%, -70%);
-  width: 50%;
-  height: 20%;
+    position: relative;
+    top: 30%;
+    left: 50%;
+    transform: translate(-50%, -70%);
+    width: 50%;
+    height: 20%;
 }
 
 #photo {
-  position: relative;
-  top: 50%;
-  left: 50%;
-  width: 200px;
-  height: 200px;
-  border-radius: 50%;
-  transform: translate(-50%, -50%);
-  background: url(photo.jpeg);
-  background-size: cover;
-  border: 2px;
-  z-index: 2;
+    position: relative;
+    top: 50%;
+    left: 50%;
+    width: 200px;
+    height: 200px;
+    border-radius: 50%;
+    transform: translate(-50%, -50%);
+    background: url(photo.jpeg);
+    background-size: cover;
+    border: 2px;
+    z-index: 2;
 }
 
 .info {
-  text-align: center;
-
-  margin-top: 10px;
-  color: #0b0329;
-  margin-bottom: 70px;
-  font: bold italic 110% serif;
+    text-align: center;
+    margin-top: 10px;
+    color: #0b0329;
+    margin-bottom: 70px;
+    font: bold italic 110% serif;
 }
 
 #menu_container {
-  position: fixed;
-  margin-left: auto;
-  margin-right: auto;
-  width: 100%;
-  height: 40px;
-  border: 2px;
-  background: #eee;
+    position: fixed;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+    height: 40px;
+    border: 2px;
+    background: #eee;
+    z-index: 2;
 }
 
 #menu {
-  list-style: none;
-  width: 800px;
-  padding: 0;
-  margin: auto;
+    list-style: none;
+    width: 800px;
+    padding: 0;
+    margin: auto;
 }
+
 #menu li {
-  float: left;
-  font: bold 14px Arial;
-  width: 20%;
+    float: left;
+    font: bold 14px Arial;
+    width: 20%;
 }
+
 #menu a {
-  color: #666;
-  display: block;
-  height: 40px;
-  line-height: 36px;
-  padding: 0px 10px 0px 10px;
-  background: #eee;
-  text-decoration: none;
+    color: #666;
+    display: block;
+    height: 40px;
+    line-height: 36px;
+    padding: 0px 10px 0px 10px;
+    background: #eee;
+    text-decoration: none;
```

Comment:
It's not a mistake, but I think it would be better to align links to the center of the container. (text-align property)

Reply:
fixed